src/CftfBundle/Entity/LsItem.php

```diff
@@ -946,6 +946,17 @@ function (LsItem $item) {
         return $ids->toArray();
     }
 
+    /*
+     * Find all children items of selected item
+     */
+    public function getDescendantIds(&$childIds)
```

Comment:
This is only gettign the child ids, I believe you wanted all descendants.

Would probably be better to return the list and do an array_merge instead of using an array reference as an argument.